jina/executors/indexers/vector/ngt.py

```diff
@@ -55,7 +55,9 @@ def query(self, keys: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple['np.nd
             results = index.search(key, size=top_k, epsilon=self.epsilon)
             index_k = []
             distance_k = []
-            [(index_k.append(result[0]), distance_k.append(result[1])) for result in results]
+            for result in results:
+                index_k.append(result[0])
```

Comment:
guess you first need to do a len(result) > 2 check, might cause `IndexError`

Reply:
I think, it would mean that the search launched an exception. Wouldn't it be too overkilling to check for every result the length? I agree though that we should have a clearer policy into sanity checks

Reply:
@nan-wang what is your thought on this?

Reply:
I didn't notice this before, bad coding style 😢 
> [(index_k.append(result[0]), distance_k.append(result[1])) for result in results]
plus, this can be as simple as
```python
index_k, distance_k = zip(*results)
```

Reply:
Changed to @hanxiao suggestion!